client/src/js/components/bhDateInterval.js

```diff
@@ -68,14 +69,13 @@ function bhDateInterval(moment, bhConstants, Fiscal, Session) {
         });
     }
 
-    $ctrl.pickerToOptions = { showWeeks : false, minDate : $ctrl.dateFrom };
-
     startup();
   };
 
   function search(selection) {
     $ctrl.selected = selection.translateKey;
     selection.fn();
+    $ctrl.pickerToOptions.minDate = $ctrl.dateFrom;
```

Comment:
This partially addresses the problem with bhDateInterval, but we will still run into issues if someone manually updates the start date.  I think the full solution (which doesn't pass tests yet) is in https://github.com/IMA-WorldHealth/bhima/pull/3632.